### PR TITLE
Fileshare userguide

### PR DESCRIPTION
**Pull request by Shruthi-1MN**

### CODE REVIEW

File: readthedocs/starterguides/userguides/fileshare_userguide.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+export OS_USER_DOMAIN_ID=default
+
+osdsctl pool list # Check if the pool resource is available
+```
+
+#### Profile creation for fileshare
+* Create profile for fileshare
+
+```yaml
+osdsctl profile create '{"name":"test_profile", "description":"test policy for fileshare", "storageType":"file"}'
```
Comment (leonwanghui) [inline readthedocs/starterguides/userguides/fileshare_userguide.md]: I think this command is duplicate in this doc, please have a check.

Comment (Shruthi-1MN) [reply readthedocs/starterguides/userguides/fileshare_userguide.md]: Yes i removed duplication.

File: readthedocs/starterguides/userguides/fileshare_userguide.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+### OpenSDS UI
+OpenSDS UI dashboard is available at `http://{your_host_ip}:8088`, please login the dashboard using the default admin credentials: `admin/opensds@123`. Create `tenant`, `user`, and `profiles` as admin.
+
+Logout of the dashboard as admin and login the dashboard again as a non-admin user to manage storage resource:
+
+#### File Share Services
+* Create a file share
+
+When you login to create file share is available in Resource tab->file share->create will allow you to create file share
```
Comment (kumarashit) [inline readthedocs/starterguides/userguides/fileshare_userguide.md]: Can we rephrase this line? 
```suggestion
Post login into the OpenSDS UI, File share can be created from: Resource -> File share -> Create
```

(Rephrase above line as per you)

File: readthedocs/starterguides/userguides/fileshare_userguide.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+![Fileshare create menu](https://github.com/Shruthi-1MN/documentation/blob/fileshare-userguide/readthedocs/starterguides/userguides/fileshare_menu.PNG)
```
Comment (kumarashit) [inline readthedocs/starterguides/userguides/fileshare_userguide.md]: Once this PR is merged, above URL will work?

File: readthedocs/starterguides/userguides/fileshare_userguide.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+* Create snapshot
+
+Click on file share name, you can find create under snapshot menu, to create snapshot
+another quick option file share resource menu->file share name-> in same row you can find Create Snapshot link to create snapshot
```
Comment (kumarashit) [inline readthedocs/starterguides/userguides/fileshare_userguide.md]:
```suggestion
Can we break it into steps:
a) Click on File Share name
b) Under Snapshot menu, click on 'Create'

Also the quick option should be another sub-section here
```

File: readthedocs/starterguides/userguides/fileshare_userguide.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+* Set access permission on file share (IP based access permissions are allowed)
```
Comment (kumarashit) [inline readthedocs/starterguides/userguides/fileshare_userguide.md]: What are access permissions? Why one need to set Access permissions? Can we add them?

File: readthedocs/starterguides/userguides/fileshare_userguide.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+Click on file share name, you can find create under access menu to create access
+another quick option file share resource menu->file share name-> in same row you can find Create Access link to create access
+
+![Fileshare snapshot and access](https://github.com/Shruthi-1MN/documentation/blob/fileshare-userguide/readthedocs/starterguides/userguides/snapshot_access.PNG)
+
+### Some file share standards and limitations
+
+* File share with the same name is not allowed to create
```
Comment (kumarashit) [inline readthedocs/starterguides/userguides/fileshare_userguide.md]:
```suggestion
* File share with the same name cannot be create
```

File: readthedocs/starterguides/userguides/fileshare_userguide.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+* File share should start with alphanumeric, special characters are not allowed
```
Comment (kumarashit) [inline readthedocs/starterguides/userguides/fileshare_userguide.md]: Break it into two line.
```suggestion
* File share can contain alphanumeric characters. Special characters are not allowed
```

File: readthedocs/starterguides/userguides/fileshare_userguide.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+* Snapshot with snapshot name is not allowed to create
```
Comment (kumarashit) [inline readthedocs/starterguides/userguides/fileshare_userguide.md]: This line is vague. Could not understand it

File: readthedocs/starterguides/userguides/fileshare_userguide.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+* Access permission set is allowed only by client IP's
+* Put operations of ACL is not permitted for now
```
Comment (kumarashit) [inline readthedocs/starterguides/userguides/fileshare_userguide.md]: Please elaborate above two lines

**Pull request merged**